WebAPI/LearningHub.Nhs.API/Controllers/ContentServerController.cs

```diff
@@ -75,9 +75,9 @@ public IActionResult GetContentDetailsByExternalReference(string externalReferen
         /// <returns>The <see cref="Task{IActionResult}"/>.</returns>
         [HttpPost]
         [Route("LogResourceReferenceEvent")]
-        public async Task<IActionResult> LogResourceReferenceEventAsync([FromBody] ResourceReferenceEventViewModel resourceReferenceEventViewModel)
+        public IActionResult LogResourceReferenceEventAsync([FromBody] ResourceReferenceEventViewModel resourceReferenceEventViewModel)
```

Comment:
I was anticipating changing the calling code to not await the response from this. I'm not sure of the effect of changing this to a synchronous task.